sql/system.go

```diff
@@ -31,7 +31,7 @@ const (
 	// MaxReservedDescID is the maximum reserved descriptor ID.
 	// All objects with ID <= MaxReservedDescID are system object
 	// with special rules.
-	MaxReservedDescID ID = 999
+	MaxReservedDescID = ID(keys.MaxReservedDescID)
```

Comment:
super incredible nit: i think `MaxReservedDescID ID = keys.MaxReservedDescID` would work because `keys.MaxReservedDescID` is an untyped constant